bin/build-metadata.js

```diff
@@ -214,6 +214,18 @@ function removeAllNumberKeys( obj ) {
 	return _.omitBy( obj, function( val, key ) { return /^\d+$/.test( key ); } );
 }
 
+function removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode( countryData ) {
+	for ( var key in countryData ) {
+		if ( countryData.hasOwnProperty( key ) ) {
+			if ( ! countryData[ key ].regionCode && ( countryData[ key ].countryDialCode + ( countryData[ key ].regionCode || '' ) ) === countryData[ key ].dialCode ) {
```

Comment:
> I must admit that it was hard to grasp this change for me - especially this condition (the countryData[ key ].regionCode || '' bit is not needed

You're right, will remove.




> How about a different approach, though: don't remove regionCode and countryDialCode :) Instead, make sure they have sane values

That was my first attempt but that won't work because

1. It increases the size of `data.js` much more.
2. It requires too many changes to switch using `countryDialCode` and `regionCode`.